src/components/StudyForm/StudyForm.tsx

```diff
@@ -34,8 +34,8 @@ const steps: Array<StepType> = [
     selector: `[data-tour="${TourId.StudyForm}-2"]`,
     content: (
       <>
-        Here, you can select a template that reflects the kind of environment
-        you&apos;ll be collecting samples from.
+        Here, you can select the templates that reflects the kind of
```

Comment:
I think there was a disagreement between the plural "templates" and the "reflects."

I also changed "kind" to "kinds" because I thought each checkbox represented one "kind" of environment, and they could now select multiple "kinds."

```suggestion
        Here, you can select the templates that reflect the kinds of
```

Reply:
Oh yeah, good catch